roadmap/1_core-protocol/upcoming/1_canister-groups/canister-groups.md

```diff
@@ -0,0 +1,13 @@
+-
+title: Canister groups
+links:
+  Forum Link: https://forum.dfinity.org/t/canister-groups/16015
+  Proposal: 
+eta:
+is_community: false
+---
+
+Communication between canisters on the same subnetwork is very fast since the messages that they exchange do not need to go through consensus. Since intercanister communication latency is less of a concern, one can scale Dapps through multi-canister architectures, as long as the canisters are guaranteed to be collocated. 
```

Comment:
`since the messages that they exchange do not need to go through consensus.` This is too easy to misunderstand and needs to be reformulated. This sounds like a single replica decides what happens.

Reply:
Maybe it is sufficient to just state the fact that communication on the same subnet is faster than between subnets – all that is needed for the motivation of this feature is *that* this is the case, not necessarily why.

Reply:
implemented the suggestion to remove the mentioning of consensus and just state the fact.